24.healthchecks/Dockerfile.elasticsearch

```diff
@@ -0,0 +1,5 @@
+FROM elasticsearch
+
+COPY elasticsearch-healthcheck /usr/local/bin/
+
+HEALTHCHECK --interval=30s --timeout=10s --start-period=30s CMD ["elasticsearch-healthcheck"]
```

Comment:
@mlocher instead of depending on the prebuilt 'healthcheck/elasticsearch' image, I figured it would be better to provide an example of how you would do it from scratch